README.md

```diff
@@ -20,8 +20,8 @@ The Learning Management System (LMS) is a web application that facilitates onlin
 - **React Router**: For client-side routing.
 - **Redux**: For state management.
 - **SASS**: Used for styling components.
-- **Webpack**: Bundles frontend assets.
-- **PostgreSQL**: As the database.
+- **Vite**: Bundles frontend assets.
+- **PostgreSQL**: Powerful relational database management system.
```

Comment:
We're not writing definition of DB provider but rather what it is used in the project?